app/uk/gov/hmrc/crsfatcaregistration/connectors/RegistrationWithUtrConnector.scala

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.crsfatcaregistration.connectors
+
+import com.google.inject.Inject
+import uk.gov.hmrc.crsfatcaregistration.config.AppConfig
+import uk.gov.hmrc.crsfatcaregistration.models.RegisterWithID
+import uk.gov.hmrc.http.{HeaderCarrier, HttpClient, HttpResponse}
+
+import scala.concurrent.{ExecutionContext, Future}
+
+class RegistrationWithUtrConnector @Inject() (
```

Comment:
I don't think there's a need for a new connector. The existing RegistrationConnector can already take a RegisterWithID and return a ETMP response.

Reply:
I think made a new one because we discuss adding the new endpoint separate from the existing one and had built it up - happened to look exactly the same 😆 
can bin this now 👍